modules/getting-started/pages/hargrave-writing-sample.adoc

```diff
@@ -0,0 +1,43 @@
+= Transportation to Salesforce Tower
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:noindex:
+
+In San Francisco, you have several transportation options for getting to Salesforce Tower:
+
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#rideshare-service[Rideshare service]
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#bus[Bus]  
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#personal-vehicle[Personal vehicle] (car, motorcycle, bicycle)
+
+== Rideshare service
```

Comment:
Why switch to sentence case for 2nd level heads? (just curious!)

Reply:
@Mulesoft-Mysti Hi - When I googled Salesforce Tower, it seemed to be capitalized that way. My intention was to use sentence case for all headings. Thanks for the quick feedback!